example/serialized_request/README.md

```diff
@@ -0,0 +1,44 @@
+# A Guide on How to Execute a Serialized Request in Client SDKs
+This guide briefly explains how to execute a serialized request in client SDKs by looking at how Node SDK can execute a request created by Web SDK.
+
+## Prerequisite
+The diagram below shows the whole architecture of this guide.
+
+![](img/samplecode.png)
+
+Web client is the client side (for example browser or mobile devices) of a product/service that is
+integrated with ScalarDLT, normally we would serialize the requests into one binary array (Uint8Array)
+and send to a designated backend service (Web service). Web service is the backend of the product/service where all the
+business logic are executed here. In this guide, the web service would just forward the serialized
+binary requests from the client side to the scalarDLT server. Therefor the web service and ScalarDLT
+server has to be up and running.
+Refer here on how to [run a ScalarDLT server](https://scalardl.readthedocs.io/en/latest/) .
+
+`Web-client` and `Web-service` Scalar DL SDKs are initialized as follows:
+```
+// web-client
+const webClientProperties = {
+    'scalar.dl.client.cert_holder_id': `foo@`,
+    'scalar.dl.client.cert_version': 1,
+    'scalar.dl.client.private_key_pem': '...',
+    'scalar.dl.client.cert_pem': '...',
+  };
+const webClient = new Scalar.ClientService(webClientProperties);
+
+// web-service
+const properties = {
+  'scalar.dl.client.server.host': '127.0.0.1',
+  'scalar.dl.client.server.port': 50051,
+  'scalar.dl.client.server.privileged_port': 50052,
+};
+const webService = new ClientServiceWithBinary(properties);
+```
+
+## Run the sample
```

Comment:
Maybe I asked the same thing before, and I think it should describe how to create a request in client side and how to proxy the request in the backend to Scalar DLT, instead of explaining how to run the sample.
Please note that what users of this document want to know is how to create a request and proxy, not how to run the sample (of-course they will run after understanding what it is).